### PR TITLE
Improve popover tour show logic to handle cases where items are …

### PR DESCRIPTION
**Pull request by mbrambilla**
…not present/visible

### CODE REVIEW

**Comment by mbrambilla** [comment]
Added `scrollIntoView()` to both popovers and tooltips to try and move the tip/dialog into view after it is shown.  Fixes the case where the tip/dialog may be offscreen, such as on mobile with the tour tooltip, but mobile is still poorly supported with these large popovers.

File: src/shared/templates/shared/partial/sidebar_bottom_end.html [hunk 1]
```diff
@@ -5,8 +5,8 @@
             data-clusive-tip-action="tour"
             onclick="tour.chain('#tour_{{ tours|list_index:0 }}');"><span class="sr-only">Start tour of features</span></button>
```
Comment (bgoldowsky) [inline src/shared/templates/shared/partial/sidebar_bottom_end.html]: Should this be tour.chainStart() ? Clicking the button as is just throws an error for me.

Comment (mbrambilla) [reply src/shared/templates/shared/partial/sidebar_bottom_end.html]: Ah, merge error on my part, pushing a fix

**Comment by bgoldowsky** [comment]
Thanks for the fix!
Working well on dashboard and library, but I am getting errors invoking the tour on Wordbank or Manage (as teacher).

**Pull request merged**